Northstar.Custom/mod/scripts/vscripts/gamemodes/sh_gamemode_gg.gnut

```diff
@@ -37,6 +37,7 @@ void function CreateGamemodeGG()
 	AddPrivateMatchModeSettingArbitrary( "#GAMEMODE_gg", "gg_kill_reward", "1.0" )
 	AddPrivateMatchModeSettingArbitrary( "#GAMEMODE_gg", "gg_assist_reward", "0.0" )
 	AddPrivateMatchModeSettingArbitrary( "#GAMEMODE_gg", "gg_execution_reward", "1.0" )
+	AddPrivateMatchModeSettingEnum( "#GAMEMODE_gg", "gg_heal_attacker", [ "#SETTING_DISABLED", "#SETTING_ENABLED" ], "0" )
```

Comment:
Should the disabled/enabled strings be the other way around? We want it to be enabled by default right?

(also the `0` might need to be a `1`)

Reply:
good point I just copy pasted sth